src/olympia/addons/indexers.py

```diff
@@ -176,6 +188,12 @@ def extract_document(cls, obj):
                 'min': min_, 'min_human': min_human,
                 'max': max_, 'max_human': max_human,
             }
+        data['author'] = [{'name': a.name, 'id': a.id, 'username': a.username}
```

Comment:
I'm not sure I like the naming. `author` implies only one object, yet it's a list of objects. Can't we just name it `listed_authors` too? Or do you plan on just removing `authors` and switch over?